config/config_defaults.go

```diff
@@ -755,7 +755,10 @@ const CGRATES_CFG_JSON = `
 
 "apier": {	
 	"caches_conns":[						// connections to CacheS for reloads
-		{"address": "127.0.0.1:2012", "transport": "*json"},
+		{"address": "*internal"},
+	],
+	"scheduler_conns": [					// connections to SchedulerS for reloads
```

Comment:
please rename scheduler_conns to schedulers_conns so we can be consistent with naming